src/AstarteManager.tsx

```diff
@@ -70,30 +70,6 @@ type Session = {
 
 const SESSION_VERSION = 1;
 
-function saveSession(session?: Session | null): void {
-  if (!session) {
-    localStorage.removeItem('session');
-  } else {
-    localStorage.setItem('session', JSON.stringify({ ...session, _version: SESSION_VERSION }));
-  }
-}
-
-function loadSession(): Session | null {
-  let session: Session | null = null;
-  try {
-    session = JSON.parse(localStorage.getItem('session') || '');
-  } catch {
-    session = null;
-  }
-  if (session && _.get(session, '_version') === SESSION_VERSION) {
-    const tokenValidation = AstarteToken.validate(session.token);
-    if (tokenValidation === 'valid') {
-      return _.omit(session, '_version');
-    }
-  }
-  return null;
-}
```

Comment:
Ideally, we want to only edit those 2 functions. This way we change how the cookie is stored/retrieved, but not how it's handled.

Reply:
token validation (`const tokenValidation = AstarteToken.validate(session.token);`) has been moved to line 94-95. check for session is a little bit redundant in old code.
Also, saveSession only saved to localStorage, this is now handled by useCookies (before, setSession was used to update react state, then saveSession for actual browser session) 
Something that I don't fully get is SESSION_VERSION, cause it is hardcoded (row 71), so basically an useless check? we can talk about that tomorrow